osbenchmark/benchmark.py

```diff
@@ -440,14 +440,14 @@ def add_workload_source(subparser):
         "--test-procedure",
         help=f"Define the test_procedure to use. List possible test_procedures for workloads with `{PROGRAM_NAME} list workloads`.")
     test_execution_parser.add_argument(
-        "--provision-config-instance",
-        help=f"Define the provision_config_instance to use. List possible "
-        f"provision_config_instances with `{PROGRAM_NAME} list "
-        f"provision_config_instances` (default: defaults).",
+        "--cluster-config",
+        help=f"Define the cluster_config to use. List possible "
```

Comment:
Nit: Update `cluster_config` --> `cluster-config` to keep consistency in help texts